addons/token-exchange/agent_mirrorpeer_delete.go

```diff
@@ -0,0 +1,139 @@
+package addons
```

Comment:
we don't need a separate file for "delete". Move it to `agent_mirrorpeer_controller.go`.